net-mail/vqadmin/vqadmin-2.3.7-r1.ebuild

```diff
@@ -11,9 +11,9 @@ SRC_URI="http://www.inter7.com/vqadmin/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-WEBAPP_MANUAL_SLOT="yes"
 KEYWORDS="~amd64 ~sparc ~x86"
-IUSE=""
+
+WEBAPP_MANUAL_SLOT="yes"
```

Comment:
That looks like something we would normally put directly atop the `inherit` line, even if `webapp.eclass` completely lacks documentation in that regard.

Also, check (and fix) HOMEPAGE.